.github/workflows/lint.yml

```
@@ -26,3 +26,1591 @@ jobs:
+
+    -   id: flake8
+
+-   repo: https://github.com/pre-commit/mirrors-mypy
+
+    rev: v1.17.1
+
+    hooks:
+
+    -   id: mypy
+
+‎CHANGELOG.md
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)